frontend/src/components/Playback/Playback.js

```diff
@@ -216,6 +301,10 @@ const Playback = ({
         }
     };
 
+    if (!sectionsLoaded) {            
+        return <Loading loadingText={'loading'} />;
+    }
+    
```

Comment:
I think this will lose us the option to set the preload message and countdown. Perhaps use the `Preload` component here instead? Also, any text should always come from the backend, so it can be localized.